docs/user-guide/administration/design-table.md

```diff
@@ -19,220 +19,257 @@ Before proceeding, please review the GreptimeDB [Data Model Documentation](/user
 
 **Cardinality**: Refers to the number of unique values in a dataset. It can be classified as "high cardinality" or "low cardinality":
 
-- **Low Cardinality Example**: Order statuses like "Pending Payment/Completed Payment/Shipped/Completed"
-  have about 4-5 unique values.
-  Days of the week are fixed at 7,
-  and the number of cities and provinces is also limited.
-- **High Cardinality Example**: User IDs can range from millions to tens of millions.
-  Device IP addresses and product SKUs are other examples of high cardinality data.
+- **Low Cardinality**: Low cardinality columns usually have constant values.
+  The total number of unique values usually no more than 10 thousand.
+  For example, `namespace`, `cluster`, `http_method` are usually low cardinality.
+- **High Cardinality**: High cardinality columns contain a large number of unique values.
+  For example, `trace_id`, `span_id`, `user_id`, `uri`, `ip`, `uuid`, `request_id`, table auto increment id are usually high cardinality.
 
 ## Column Types and Selection
 
-In GreptimeDB, columns are categorized into three types: Tag, Field, and Time Index.
-The timestamp usually represents the time of data sampling or the occurrence time of logs/events and is used as the Time Index column.
-GreptimeDB optimizes data organization based on the Time Index to enhance query performance.
+In GreptimeDB, columns are categorized into three semantic types: `Tag`, `Field`, and `Timestamp`.
+The timestamp usually represents the time of data sampling or the occurrence time of logs/events.
+GreptimeDB uses the `TIME INDEX` constraint to identify the `Timestamp` column.
+So the `Timestamp` column is also called the `TIME INDEX` column.
+If you have multiple columns with timestamp data type, you can only define one as `TIME INDEX` and others as `Field` columns.
 
-### Tag Columns
+In GreptimeDB, tag columns are optional. The main purposes of Tag columns include:
 
-Tag columns, also known as label columns,
-generally carry metadata of the measured data or logs/events.
-For example, when collecting nationwide weather temperature data,
-the city (e.g., `city="New York"`) is a typical tag column.
-In monitoring, system metrics like CPU and memory usually involve a `host` tag to represent the hostname.
+1. Defining the ordering of data in storage.
+   GreptimeDB reuses the `PRIMARY KEY` constraint to define tag columns and the ordering of tags.
+   Unlike traditional databases, in GreptimeDB, primary key and time series are identical.
+   Tables in GreptimeDB sort rows by (primary key, timestamp).
+   This improves the locality of data with the same tags.
+   If there are no tag columns, GreptimeDB sorts rows by timestamp.
+2. Identifying a unique time-series.
+   When the table is not append-only, GreptimeDB can deduplicate rows by timestamp under the same time-series (primary key).
+3. Smoothing migration from other TSDBs that use tags or labels.
```

Comment:
We need to mention the impact of the order of tags in primary key

Reply:
The `When to use primary key` section explains this and has an example.